apps/client/src/utils/test-ids.js

```diff
@@ -1,42 +1,49 @@
-export const LOCATION_FOUND = "location-found";
+// Navigation
```

Comment:
Nice, thanks for organising this file! 👍

Reply:
You are welcome my friend.